tasks/RedHat.yml

```diff
@@ -12,7 +12,7 @@
 - name: Download package locally
   become: no
   connection: local
-  get_url: "url={{ couchbase_server_centos_ee_url }} dest={{ role_path }}/files/{{ couchbase_server_centos_ee_package }} sha256sum={{ couchbase_server_centos_ee_sha256 }} timeout=240"
+  get_url: "url={{ couchbase_server_centos_ee_url }} dest={{ role_path }}/files/{{ couchbase_server_centos_ee_package }} mode=0644 sha256sum={{ couchbase_server_centos_ee_sha256 }} timeout=240"
```

Comment:
You can change the umask for the user:

https://serverfault.com/questions/458533/setting-umask-for-all-users
https://unix.stackexchange.com/questions/86760/how-to-change-default-permissions-of-files-created-by-a-process